src/convertToObject.js

```diff
@@ -12,7 +12,34 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+
+  const arrayStyles = styles.map(style => style.split(':'));
+  const object = {};
```

Comment:
[checklist 2](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md#:~:text=%5BCODE%20STYLE%5D%20%2D%20Avoid%20variable%20namings%20like%20array%20%2C%20object%20arr%2C%20obj%2C%20result.%20It%20will%20decrease%20code%20readability)

Reply:
fix this moment